src/sentry/static/sentry/app/components/forms/selectControl.tsx

```diff
@@ -292,14 +314,30 @@ const SelectControl = props => {
       {...rest}
     />
   );
-};
+}
 SelectControl.propTypes = SelectControlLegacy.propTypes;
 
 const SelectControlWithTheme = withTheme(SelectControl);
 
-const SelectPicker = ({async, creatable, forwardedRef, ...props}) => {
+type PickerProps = ControlProps & {
+  /**
+   * Enable async option loading.
+   */
+  async?: boolean;
+  /**
+   * Enable 'create' mode which allows values to be created inline
```

Comment:
```suggestion
   * Enable 'create' mode which allows values to be created inline.
```

just to be consistent :)